client/src/components/EventForm/ShowEvents/ShowOneEvent.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    width: "100%",
+  },
+  bullet: {
+    display: 'inline-block',
+    margin: '0 2px',
+    transform: 'scale(0.8)',
+  },
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    margin: ".5rem auto"
+  },
+  small: {
+    fontSize: ".8rem",
+    color: "grey",
+  },
+  body: {
+    fontSize: "1.2rem",
+    color: "grey",
+    margin: ".5rem auto"
+  },
+  labels: {
+      fontSize: ".8rem",
+      fontWeight: "bolder",
+      left: "0",
+      textAlign: "left",
+      marginBottom: ".5rem"
+  },
+  values: {
+    fontSize: ".8rem",
+    fontWeight: "lighter",
+    left: "0",
+    testAlign: "center",
+    marginBottom: "1.5rem"
+  }
+});
+
+
+function ShowOneEvent(props) {
+    const classes = useStyles()
```

Comment:
Use object destructuring on props to assign variables.

Reply:
OK

Reply:
> Use object destructuring on props to assign variables.

@07souravkunda done, Review it again plz,